catalog.c

```diff
@@ -266,7 +266,7 @@ catalog_get_backup_list(const pgBackupRange *range)
  * Find the last completed database full valid backup from the backup list.
  */
```

Comment:
How about adding a comment so that you can specify that the problem still exists?
Check the already registered issue #154, it will be helpful to add comments on the remaining issues.